src/Course/FileIO.hs

```diff
@@ -81,50 +82,50 @@ the contents of c
 
 -- Given the file name, and file contents, print them.
 -- Use @putStrLn@.
-printFile ::
-  FilePath
-  -> Chars
-  -> IO ()
-printFile =
-  error "todo: Course.FileIO#printFile"
+printFile :: FilePath -> Chars -> IO ()
+printFile path chars = putStrLn ("The path: " ++  path ++ ". The chars: " ++ chars)
 
 -- Given a list of (file name and file contents), print each.
 -- Use @printFile@.
-printFiles ::
-  List (FilePath, Chars)
-  -> IO ()
-printFiles =
-  error "todo: Course.FileIO#printFiles"
+printFiles :: List (FilePath, Chars) -> IO ()
+printFiles = voidSequenceMap $ uncurry printFile
```

Comment:
beautiful

Reply:
I wonder if it's the same as `voidSequenceMap . uncurry printFile`

Reply:
I think it is not since that's equal to `printFiles = voidSequenceMap (uncurry printFile)` right? (I tried it btw, and it doesn't compile)

Reply:
yeah what you wrote is `voidSequenceMap (uncurry printFile)`, whereas with composition it would be `voidSequenceMap . (uncurry printFile)` so that's obvious not the same thing :)